client/flutter/lib/l10n/your_questions_answered.yaml

```
@@ -0,0 +1,35 @@
+schema_version: 1
+contents:
+  type: qa
```

Not a blocker, but consider making this more descriptive/generic, e.g. `accordionlist` or `questionanswer`. I'm just not sure someone looking at this down the line would interpret `qa` as question-answer. Since we're not using `type` anywhere yet anyway, it's totally up to you whether you want to change here and what you'd like to use.

```suggestion
  type: accordionlist
```

---

I was thinking of this as identifying the type of content and not the desired presentation...  I think we will only have a few of these (maybe two) and the content itself could be presented in different ways...

---

makes sense re type of content, I think the comment was more that `qa` might not be immediately obvious to folks that it's question-answer. not a blocker on this PR